cmd/webhook/main.go

```diff
@@ -214,10 +216,16 @@ func NewConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 }
 
 func main() {
+	// Read the port number of webhook
+	port, err := strconv.Atoi(os.Getenv("WEBHOOK_PORT"))
+	if err != nil {
+		port = 8443
+	}
```

Comment:
If we do this, we should adopt it consistently across serving / eventing / eventing-contrib.

I suspect a helper like `system.Namespace()` would be useful, maybe `webhook.Port(8443)`?

Reply:
Sounds good. I will open a PR in knative/pkg to add the helper function. Thank you for the advice.

Reply:
https://github.com/knative/pkg/pull/1162 for the PR in pkg.

Reply:
FInally merged :)